common/fileset-utils/src/main/resources/xml/xproc/fileset-add-entry.xpl

```diff
@@ -1,11 +1,55 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<p:declare-step version="1.0" type="px:fileset-add-entry" name="main" xmlns:p="http://www.w3.org/ns/xproc" xmlns:d="http://www.daisy.org/ns/pipeline/data"
-  xmlns:px="http://www.daisy.org/ns/pipeline/xproc" exclude-inline-prefixes="px">
+<p:declare-step xmlns:p="http://www.w3.org/ns/xproc" version="1.0"
+                xmlns:px="http://www.daisy.org/ns/pipeline/xproc"
+                xmlns:d="http://www.daisy.org/ns/pipeline/data"
+                exclude-inline-prefixes="px"
+                type="px:fileset-add-entry" name="main">
 
-  <p:input port="source"/>
-  <p:output port="result"/>
+  <p:documentation xmlns="http://www.w3.org/1999/xhtml">
+    <p>Add a new entry to a fileset.</p>
+  </p:documentation>
+
+  <p:input port="source" primary="true">
+    <!-- FIXME: rename to source.fileset -->
```

Comment:
Oops didn't intend to include this comment in the commit. You're right, because we're updating the major version number we could do it now. But I was thinking about doing a global rename of the ports anyway so we could also cover it then. The major version update ([Change behavior of px:fileset-copy and px:fileset-move](3453309b558b42bd15ad7fea989eafebb4fcac15)) was done after this change ([Add in-memory ports to px:fileset-add-entry](494500cb0e574e4873dd3873d2ee6d9a919450f0)) so I probably didn't even realize it could be done now.